pkg/iac/rego/load.go

```diff
@@ -127,9 +144,73 @@ func (s *Scanner) LoadPolicies(enableEmbeddedLibraries, enableEmbeddedPolicies b
 	return s.compilePolicies(srcFS, paths)
 }
 
+func (s *Scanner) fallbackChecks(compiler *ast.Compiler) {
+
+	var excludedFiles []string
+
+	for _, e := range compiler.Errors {
+		loc := e.Location.File
```

Comment:
@nikpivkin @simar7 I have bump trivy-operator with latest trivy `0.51.1` and I'm getting `nil pointer` exception on ***L152*** while running my policies tests
`loc := e.Location.File` anyway to protect it?

here is [link](https://github.com/aquasecurity/trivy-operator/actions/runs/8957717099/job/24601151551?pr=2054) for test failure

Reply:
Location can be nil. I'll add a check.

Reply:
@chen-keinan Opened the PR with a fix. https://github.com/aquasecurity/trivy/pull/6638